examples/src/main/resources/more_people.json

```diff
@@ -0,0 +1,3 @@
+{"name":"Rob Stark", "address":{"city":"Columbus","state":"Ohio"}}
+{"name":"Ned Stark", "address":{"city":"San Diego", "state":"California"}}
+{"name":"Jon Snow", "address":{"city":"San Diego", "state":"California"}}
```

Comment:
Last known cities of Rob, Ned and Jon are WinterFell, KingsLanding and The Wall respectively. Please correct. As a suggestion, it will be good to have The Lannisters in the some_people.json. :-)